test/graphql/udt_test.exs

```diff
@@ -139,4 +139,48 @@ defmodule GodwokenExplorer.Graphql.UDTTest do
              json_response(conn, 200)
            )
   end
+
+  test "graphql: udt holders ", %{
```

Comment:
update on native token is triggered by its bridged one should be covered in the test

Reply:
Has been added